samples/nrf9160/modem_shell/src/link/link_shell.c

```diff
@@ -107,10 +109,12 @@ static const char link_usage_str[] =
 	"                supporting RAI. Effective when going to normal mode.\n";
 
 static const char link_settings_usage_str[] =
-	"Usage: link settings --read | --reset\n"
+	"Usage: link settings --read | --reset | --mreset_all | --mreset_user\n"
 	"Options:\n"
-	"  -r, --read,   Read and print current persistent settings\n"
-	"      --reset,  Reset all persistent settings as their defaults\n";
+	"  -r, --read,        Read and print current persistent settings\n"
+	"      --reset,       Reset all persistent settings as their defaults\n"
+	"      --mreset_all,  Reset all modem settings as their defaults\n"
+	"      --mreset_user, Reset modem user configurable settings as their defaults\n";
```

Comment:
"Reset all persistent settings to defaults" and same for other lines.

Reply:
Original `--reset` hook is having: "Reset all persistent settings as their defaults"
I recall that it was pretty much like you proposed but this current was something that somebody wanted it to be in some previous review and these new ones are following that ;-)

Reply:
Changed.